tests/unit/test_settings_health.py

```
@@ -0,0 +1,169 @@
+
+        rt = MagicMock()
+        rt.get_health_snapshot.return_value = snapshot
+
+        with client.application.app_context():
+            client.application.config["REFRESH_TASK"] = rt
+
+        resp = client.get("/api/health/plugins")
+        data = resp.get_json()
+        assert "fresh_plugin" in data["items"]
+
+    def test_keeps_entries_without_last_seen(self, client, monkeypatch):
```

Ruff (0.15.7)

[warning] Unused method argument: `monkeypatch`

(ARG002)